Salary App/ViewController.swift

```diff
@@ -66,33 +70,33 @@ class ViewController: UIViewController, UITextFieldDelegate {
             let salary = Int(self.salaryTextField.text!) ?? 0
```

Comment:
When working with money its better to use decimals, with int's it can only ever by a whole number what about if i get paid £500.50 using an Int will turn this into 500 which isn't correct